### ISSUE TITLE
Kiribati (1.7.a) View vaccination events in Mobile - backfill

### ISSUE THREAD

**Issue by adamdewey** [openmsupply/mobile#4650]
## Is your feature request related to a problem? Please describe.

#4623 adds the ability to display vaccination event data in Mobile. 

Vaccination events created after #4623 is deployed will be displayed correctly, however old data is incompatible and needs to be updated. 

This ticket is to design a data script to update the existing vaccination event data so that it can be picked up by the new display functionality added in #4623 

## Describe the solution you'd like

**A clear and concise description of what you want to happen.**

## Implementation

**A description of how the solution would best be implemented in the code base**

## Describe alternatives you've considered

**A clear and concise description of any alternative solutions or features you've considered.**

## Additional context

**Add any other context or screenshots about the feature request here.**

**Comment by adamdewey** [comment]
Duplicate of #4638
